setup.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+"""chaostoolkit dynatrace extension builder and installer"""
+
+import sys
+import io
+
+import setuptools
+
+sys.path.insert(0, ".")
+from chaosdynatrace import __version__
+sys.path.remove(".")
+
+name = 'chaostoolkit-dynatrace'
+desc = 'Chaos Toolkit dynatrace support'
+
+with io.open('README.md', encoding='utf-8') as strm:
+    long_desc = strm.read()
+
+classifiers = [
+    'Development Status :: 2 - Pre-Alpha',
+    'Intended Audience :: Developers',
+    'License :: Freely Distributable',
+    'Operating System :: OS Independent',
+    'License :: OSI Approved :: Apache Software License',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.5',
+    'Programming Language :: Python :: 3.6',
+    'Programming Language :: Python :: Implementation',
+    'Programming Language :: Python :: Implementation :: CPython'
+]
+author = 'chaostoolkit Team'
+author_email = 'sh@defuze.org'
```

Comment:
I'll change this to contact@chaostoolkit.org

But you can also use your own email if you prefer since you authored the code?

Reply:
Hello @Lawouach,
the email contact@chaostoolkit.org is perfect.